src/python/pants/notes/2.5.x.md

```diff
@@ -2,6 +2,72 @@
 
 See https://www.pantsbuild.org/v2.5/docs/release-notes-2-5 for an overview of the changes in this release series.
 
+## 2.5.0.dev1 (Apr 10, 2021)
+
+### New Features
+
+* Add dependency inference for Shell ([#11857](https://github.com/pantsbuild/pants/pull/11857))
+
+* Add Shell support with Shellcheck, shmft, and shunit2 test runner ([#11844](https://github.com/pantsbuild/pants/pull/11844))
+
+* Warn if config files detected but not wired up to Pants ([#11852](https://github.com/pantsbuild/pants/pull/11852))
+
+* Warn when dependency inference fails for `protobuf_library` targets due to ambiguity ([#11849](https://github.com/pantsbuild/pants/pull/11849))
+
+### User API Changes
+
+* Remove deprecated `unzip` field for `pex_binary` ([#11868](https://github.com/pantsbuild/pants/pull/11868))
+
+* Deprecate `--ignore-pants-warnings` in favor of `--ignore-warnings` ([#11851](https://github.com/pantsbuild/pants/pull/11851))
+
+### Bug fixes
+
+* Deflake tar extraction by setting the cwd through the engine ([#11867](https://github.com/pantsbuild/pants/pull/11867))
+
+* Hotfix build wheels jobs failing due to new Shell backend ([#11855](https://github.com/pantsbuild/pants/pull/11855))
+
+* Fix typo in `tags` help message (Cherry-pick of #11836)
```

Comment:
Hrm, a cherry pick sounds wrong since this is master. I guess @Eric-Arellano did the change on 2.3.x and then cherry-picked elsewhere?

Reply:
Oh, I just messed up the title. Note that there is only one PR number listed here.

@wisechengyi please remove the (cherry pick of) part. Also, this link is missing - see the above entry for the markdown format.

Reply:
Looking at the original PRs, it does look like the change happened directly on 2.3.x and then was cherrypicked into main. Eric, are you sure about this?

Reply:
I think this should be under `### Documentation` in the notes, and deleted from the "Internal" list in the PR description.